### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by toas-koas** [nucleus-ffm/foss_warn#90]
**Describe the bug**
After start from the phone the app is crashing

**To Reproduce**
Start the phone

**Expected behavior**
No crash

 
**Device :**
 - Device: pixel 2xl
 - OS: Android 13
 - App-version Version 0.5.1
 - Battery optimization [false]

```
type: crash
osVersion: google/lineage_taimen/taimen:13/TQ2A.230505.002/eng.emy.20230613.033103:user/release-keys
package: de.nucleus.foss_warn:27
process: de.nucleus.foss_warn

java.lang.RuntimeException: Unable to start receiver com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver: java.lang.RuntimeException: Missing type parameter.
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:4312)
	at android.app.ActivityThread.-$$Nest$mhandleReceiver(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2149)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7884)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952)
Caused by: java.lang.RuntimeException: Missing type parameter.
	at g0.a.d(Unknown Source:26)
	at g0.a.<init>(Unknown Source:7)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin$a.<init>(Unknown Source:0)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin.loadScheduledNotifications(Unknown Source:25)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin.rescheduleNotifications(Unknown Source:0)
	at com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver.onReceive(Unknown Source:38)
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:4303)
	... 9 more

```

**Comment by DocSniper** [comment]
Can confirm this issue.

OS: Android: 13 / LineageOS 20
App-version: 0.5.1 (F-Droid)

```
Process: de.nucleus.foss_warn, PID: 10537
java.lang.RuntimeException: Unable to start receiver com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver: java.lang.RuntimeException: Missing type parameter.
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:4315)
	at android.app.ActivityThread.-$$Nest$mhandleReceiver(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2152)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7918)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
Caused by: java.lang.RuntimeException: Missing type parameter.
	at g0.a.d(Unknown Source:26)
	at g0.a.<init>(Unknown Source:7)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin$a.<init>(Unknown Source:0)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin.loadScheduledNotifications(Unknown Source:25)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin.rescheduleNotifications(Unknown Source:0)
	at com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver.onReceive(Unknown Source:38)
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:4306)
	... 9 more
```

**Comment by nucleus-ffm** [comment]
Thank you for the bug report. I was able to reproduce the problem myself and also found the solution. The problem can be solved if we add some proguards rules as mentioned [here](https://github.com/MaikuB/flutter_local_notifications/issues/1490#issuecomment-1041455128) by the developer. I added the [file](https://github.com/nucleus-ffm/foss_warn/blob/bugfixes/android/app/proguard-rules.pro) and the crash seems to be fixed. A side effect seems to be that fosswarn triggers the next background task only after the defined frequency and not directly at startup. Before that, fosswarn seems to have restarted because of the crash, which also caused the background service to run directly. Maybe we can think about implementing a callback that manually triggers the background task after startup.
